tasks.php

```diff
@@ -1719,6 +1722,7 @@ function property_format_value($property_id, $task_a, $for_list_of_tasks)
     global $tasks_close_array;
     global $tasks_status_array;
     global $versions_array;
+    global $Parsedown;
```

Comment:
Why is the global here? I don't think we need it.

Reply:
whoops, leftover from refactor. Will remove